app/uk/gov/hmrc/hmrcemailrenderer/templates/itr/ItrTemplates.scala

```diff
@@ -30,7 +30,14 @@ object ItrTemplates {
       subject = "Apply to use Venture Capital Schemes: verify your email address",
       plainTemplate = txt.itrEmailVerification.f,
       htmlTemplate = html.itrEmailVerification.f,
-      priority = Some(MessagePriority.Urgent))
+      priority = Some(MessagePriority.Urgent)),
+    MessageTemplate.create(
+      templateId = "investment_tax_relief_confirmation_email",
+      fromAddress = govUkTeamAddress,
+      service = InvestmentTaxRelief,
+      subject = "Apply to use Venture Capital Schemes: Advance Assurance application received",
+      plainTemplate = txt.itrEmailConfirmation.f,
+      htmlTemplate = html.itrEmailConfirmation.f)
```

Comment:
Should this go to the Urgent queue like your other template? It might take a few hours to process something on the Standard (default) queue
Please see:
https://github.com/hmrc/hmrc-email-renderer/blob/master/CONTRIBUTING.md#volumes-and-priority-recommendations